src/widgets/APIDialog/APIDialog.js

```diff
@@ -65,6 +115,7 @@ function APIDialog() {
       type: "SAVE_API_DIALOG",
       payload: {
         method,
+        path: pathRef.current,
```

Comment:
`updatePath(apiRef.current, path, pathRef.current)` line at 61 runs for adding resource as well, and unnecessarily updating path causing the bug